README.md

```diff
@@ -34,10 +34,10 @@ For each source in this configuration file, the following globs will be expanded
 * `$SOURCE_PATH/*/PKGBUILD` will be built with [makepkg(8)](https://www.archlinux.org/pacman/makepkg.8.html).
 * `$SOURCE_PATH/*.pkg.toml` will be built with [holo-build(8)](https://github.com/holocm/holo-build).
 
-Note that directories will not be traversed any deeper than that. For example, any PKGBUILD file must be in a direct
-subdirectory of the source path specified in the configuration file.
+All desired PKGBUILD files must be in a direct subdirectory of the source path specified in the configuration file. 
+Directories will not be traversed any deeper than that. 
```

Comment:
I think this is backwards now. The change puts the emphasis on the concrete case of PKGBUILDs, while the previous wording refers to both globs equally and uses PKGBUILDs only as an example.

Reply:
Ah, Fair enough.